src/OSSupport/File.h

```diff
@@ -163,8 +163,12 @@ class cFile
 	/** Returns the list of all items in the specified folder (files, folders, nix pipes, whatever's there). */
 	static AStringVector GetFolderContents(const AString & a_Folder);  // Exported in ManualBindings.cpp
 
-	int Printf(const char * a_Fmt, fmt::ArgList);
-	FMT_VARIADIC(int, Printf, const char *)
+	int vPrintf(const char * a_Fmt, fmt::printf_args);
+	template <typename... Args>
+	int Printf(const char * a_Fmt, const Args & ... a_Args)
+	{
+		return vPrintf(a_Fmt, fmt::make_printf_args(a_Args...));
```

Comment:
mhm, you might be surprised, this actually increased my binary size by 0.1

Reply:
I tried replacing the main `Printf` override in `StringUtils.h` with a templated version and my results were:
clang - 0.1 MB increase
gcc- 0.01 MB increase

Not earth shattering but still worse than before. What compiler are you testing on?

Reply:
Oh not in StringUtils, just in this one, File.h, Visual C++.

Anyway consistency dictates this is fine.